docs/articles/faq/README.md

```diff
@@ -255,14 +255,26 @@ However, we recommend that you determine the cause of this issue and address it.
 
 This error can occur in the following situations:
 
-#### The Web server is not responding
+#### The Web server is not responding or takes too long to respond
```

Comment:
> or takes too long to respond

It's not necessary. The term `he Web server is not responding` means that the server is not responding for the allowed timeout.